x/transfermiddleware/relay_test.go

```diff
@@ -362,3 +362,105 @@ func (suite *TransferMiddlewareTestSuite) TestTimeOutPacket() {
 	chainBSenderBalances := suite.chainB.AllBalances(suite.chainB.SenderAccount.GetAddress())
 	suite.Equal(expBalance, chainBSenderBalances)
 }
+
+func (suite *TransferMiddlewareTestSuite) TestMint() {
+	var (
+		transferAmount, _ = sdk.NewIntFromString("10000000000000000000")
+		// when transfer via sdk transfer from A (module) -> B (contract)
+		timeoutHeight = clienttypes.NewHeight(1, 110)
+		path          *customibctesting.Path
+		expDenom      = "ibc/C053D637CCA2A2BA030E2C5EE1B28A16F71CCB0E45E8BE52766DC1B241B77878"
+		// pathBtoC      = NewTransferPath(suite.chainB, suite.chainC)
+	)
+
+	testCases := []struct {
+		name string
+	}{
+		{
+			"Test Mint",
+		},
+	}
+	for _, tc := range testCases {
+		suite.Run(tc.name, func() {
+			suite.SetupTest()
```

Comment:
Can we create `setup()` func ?

Reply:
Hmm, do we really need? I think it just the same. Just different amount of genesis token